text/0054-rth-payment-id-memos.md

```diff
@@ -0,0 +1,127 @@
+- Feature Name: `Recoverable Transaction History Payment ID Memos`
+- Start Date: (2022-10-18)
+- MCIP PR: [mobilecoinfoundation/mcips#0054](https://github.com/mobilecoinfoundation/mcips/pull/54)
+
+# Summary
+[summary]: #summary
+
+[#4 Recoverable Transaction History (RTH)](https://github.com/mobilecoinfoundation/mcips/pull/4) introduced three different memo types for [encrypted memos](https://github.com/mobilecoinfoundation/mcips/pull/3). This proposal specifies three more memo types:
+
+- Destination With Payment Request Id Memo
+- Authenticated Sender With Payment Id Memo
```

Comment:
Can we add the word `Intent` to these two memo names - Authenticated Sender With Payment Intent Id Memo, Destination With Payment Intent Id Memo - it will help disambiguate between a "request" and an "intent". Left simply as "Payment Id Memo" someone could conceivably mismatch with "Payment Request ID".

Reply:
Done